frontend/views/components/UsersSelector.vue

```diff
@@ -0,0 +1,189 @@
+<template lang='pug'>
+form.c-search-form(@submit.prevent='')
+  label.field
+    .sr-only {{label}}
+    .inputgroup.c-search(
+      @click='clear'
+    )
+      .is-icon.prefix(aria-hidden='true')
+        i.icon-search
+      .input(
+        type='text'
+        name='search'
+      )
+        .profile-wrapper(
+          contenteditable='false'
+          v-for='(username, index) in usernames'
+          :key='index'
+        )
+          .profile
+            avatar-user(:username='username' size='xs')
+            .c-name.has-text-bold {{ displayName(username) }}
+            .button.is-icon-small(
+              @click.prevent.stop='remove(username)'
+              :aria-label='L("Clear search")'
+            )
+              i.icon-times
+        .c-keyword(
+          contenteditable
+          ref='input'
+          data-test='users-selector'
+          @keydown='onHandleKeyDown'
+          @keyup='onHandleKeyUp'
+        )
+</template>
+
+<script>
+import { mapGetters } from 'vuex'
+import AvatarUser from '@components/AvatarUser.vue'
+
+export default ({
+  name: 'UsersSelector',
+  components: {
+    AvatarUser
+  },
+  props: {
+    label: {
+      type: String,
+      required: true
+    },
+    usernames: {
+      type: Array,
+      default: []
+    },
+    defaultValue: {
+      type: String,
+      default: ''
+    },
+    autofocus: {
+      type: Boolean,
+      default: false
+    }
+  },
+  data () {
+    return {
+      // NOTE: v-model can't be used here since it's only for limited elements; input, select, textarea
+      //       https://vuejs.org/api/built-in-directives.html#v-model
+      value: ''
```

Comment:
@Silver-IT why isn't this using an `input` element?

Reply:
Having looked the Slack, I can see it is automatically resizing and repositioning.
![20230426_070318](https://user-images.githubusercontent.com/57976479/234426753-ccbf9885-5c99-400e-8e43-2cbfcd4f63df.gif)
I don't think we can implement this using `input` or `textarea` elements.